index.js

```diff
@@ -162,23 +162,14 @@ class ExecutorQueue extends Executor {
             winston.info(`scheduler enqueuing job timestamp  >>  ${JSON.stringify(job)}`));
 
         this.multiWorker.start();
-        this.scheduler.connect(() => {
-            this.scheduler.start();
-        });
+        Promise.resolve().then(this.scheduler.connect()).then(this.scheduler.start());
```

Comment:
does `this.scheduler.connect()` returns a function ?
same question for `.start()` function. what's the need for this syntax here ?

Reply:
I will fix that. sorry